website/language.py

```diff
@@ -72,6 +72,21 @@
 
 INVALID_EMAIL_CONFIRM_TOKEN = 'This confirmation link is invalid. Please <a href="/login/">log in</a> to continue.'
 
+CANNOT_MERGE_ACCOUNTS_SHORT = 'Cannot Merge Accounts'
+
+CANNOT_MERGE_ACCOUNTS_LONG = 'Accounts cannot be merged due to a conflict.'
```

Comment:
Long: "Accounts cannot be merged due to a possible conflict with add-ons, you will want to deactivate any add-ons authorized on your to-be-merged account and try again." .... ?